src/public/app/services/note_tooltip.js

```diff
@@ -93,6 +93,10 @@ async function mouseEnterHandler() {
             }
         }, 1000);
     }
+    // When the mouse is moved out, it means that the tooltip is no longer being attended to.
+    $(`.${linkId}`).on("mouseleave", () => {
+        setTimeout(() => $(this).tooltip('dispose'), 1000);
+    })
```

Comment:
How does this differ from the code a few lines above disposing of the tooltip after a second if mouse doesn't hover over it? Seems to be doing the same thing ...